### PR TITLE
switch liquid staker from clock to op_mode based verification

### LINKED ISSUE #250: [Clock dep] Stride LSer - Remove Clock dependency
As implemented in #240, implement the concept of **privileged accounts** for either _permissioned_ or _permissionless ticking_ operation of the contract.

- [ ] Remove dependency to Clock contract, and the Enqueue message at instantiation
- [ ] Add Privileged Accounts, and a guard for the ticking operation
- [ ] Update top-level covenants (using this contract) to set its address in the Clock's `initial_queue`
- [ ] Update / add tests as needed

### PR DESCRIPTION
**Pull request by bekauz**
closes #250

### CODE REVIEW

File: contracts/single-party-pol-covenant/src/contract.rs [hunk 1]
```diff
@@ -101,10 +101,12 @@ pub fn instantiate(
     clock_whitelist.push(liquid_staker_instantiate2_config.addr.to_string());
```
Comment (stiiifff) [inline contracts/single-party-pol-covenant/src/contract.rs:101]: It could be removed from the whitelist if added to the initial queue.

**Review by stiiifff — COMMENTED** [review]
No change in integration tests ?

**Comment by bekauz** [comment]
great point, I thought it would be caught by the CI but forgot that single party e2e is not yet enabled

**Comment by bekauz** [comment]
the top level covenant interface did not change so no changes to e2e are needed, but I cleaned up a few things there nevertheless @stiiifff

**Review by stiiifff — APPROVED** [review]

**Pull request merged**